src/api/ApiAccessors.py

```
@@ -29,97 +29,91 @@ def __init__(self, route: str = "") -> None:
         self.host_config_key = "user_api"
 
 
+class DataApiResourceObject(TypedDict):
+    id: str
+    type: str
+    attributes: NotRequired[dict[str, Any]]
+    relationships: NotRequired[dict[str, DataApiResponse]]
```

Ruff (0.14.10)

Undefined name `DataApiResponse`

(F821)